annotation/app/microservice_communication/user.py

```diff
@@ -0,0 +1,45 @@
+import os
+from typing import List, Union, Dict
+
+import requests
+from dotenv import find_dotenv, load_dotenv
+from requests import RequestException
+from app.models import ManualAnnotationTask
+
+load_dotenv(find_dotenv())
+USERS_SEARCH_URL = os.environ.get("USERS_SEARCH_URL")
+
+
+class GetUserInfoException(Exception):
+    def __init__(self, exc_info: Union[str, RequestException]):
+        self.exc_info = exc_info
+
+
+def get_response(callback_url: str, user_id: str, tenant: str, token: str):
+    try:
+        user_response = requests.get(
+            f"{callback_url}/{user_id}",
+            headers={
+                "X-Current-Tenant": tenant,
+                "Authorization": f"Bearer {token}",
+            },
+            timeout=5,
+        )
+        if user_response.status_code != 200:
+            raise GetUserInfoException(user_response.text)
+        return user_response.json().get("username")
```

Comment:
I believe it is important to let API clients know, that request to 'users' microservice has failed.
I would probably use: 
raise GetUserInfoException(f"Failed request to 'users' microservice: {user_response.text}")